gaskij/rpicampusmap
# WIP: Merge Dev into master to prevent too much divergence

## Description
The `Dev` branch has been in progress for over a year and has too many changes to continue building it up. We need to clean it up and merge the good changes into master so that we can follow the new structure of using smaller feature branches in the future.

## Solution
Resolved any merge conflicts between the two branches, remove features that will not be included in the actual build, and fix any existing issues.

## Known Issues
- ~~Merge conflicts exist between the two branches~~ merged master into Dev and pushed
- ~~RPI Seal should be removed from footer and reduced in height so that the map screen doesn't scroll~~ done
- Landing page should be removed and map screen should be the first page on site visit
- Warnings importing custom angular controllers across multiple pages
- Revert to original page styles
- Lint errors
- Info pages do not work
    - ~~Routing returns a 404~~ fixed
    - Info does not populate page
- Search page does not work
    - Does not return results for given queries
    - `Search results ""` should show given query, not number of results

## Testing Procedures
- Try out all the pages and make sure everything is working properly (buttons, header, links, etc.)
- Resize the window and ensure things are still correct.
- Make sure there are no errors in the console.

## Checklist for author
